ru/_version.py

```diff
@@ -1 +1,4 @@
-__version__ = '0.0.4'
+"""version.py
+Version of the read until software
+"""
+__version__ = "0.0.5a4"
```

Comment:
Shouldn't this be a new version now?

Reply:
We should probably bump this to 0.0.6 or something along those lines

Reply:
Yep.